src/current/_includes/releases/v23.1/v23.1.28.md

```diff
@@ -3,6 +3,11 @@
 Release Date: October 10, 2024
 
 {% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v23-1-28-security-changes">Security changes</h3>
+
+ [`SHOW CHANGEFEED JOB`]({% link v23.1/show-changefeed-job.md %}), [`SHOW CHANGEFEED JOBS`]({% link v23.1/show-changefeed-jobs.md %}#show-changefeed-jobs), and [`SHOW JOBS`]({% link v23.1/show-jobs.md %}) commands no longer expose user sensitive information like `client_key`. [#129910][#129910]
```

Comment:
there is no page `v23.1/show-changefeed-job.md`, please remove link.

Reply:
Fixed, Kathryn asked me to link both commands to the same section.